srcpkgs/kdiff3/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'kdiff3'
+pkgname=kdiff3
+version=0.9.98
+revision=1
+build_style=gnu-configure
+#build_wrksrc=dist
+makedepends="qt-qmake qt-devel qt-devel-tools"
+short_desc="Both kdiff and kdiff2 name taken"
```

Comment:
this seems like a really bad description

Reply:
The description was taken from the original project FAQ. I can make it "generic" in next patch if absolutely necessary.

Reply:
Please make it descriptive of what the program does. see the Arch Linux example.